src/emulator/constants.ts

```diff
@@ -5,6 +5,7 @@ import { Address, Emulators } from "./types";
 const DEFAULT_PORTS: { [s in Emulators]: number } = {
   gui: 4000,
   hub: 4400,
+  logging: 9999,
```

Comment:
nit: Can we keep this down in the 4000s with the other administrative stuff?  Maybe `4500` as default so that its outside the search range of `hub`.